Google.PowerShell.IntegrationTests/BigQuery/BigQuery.BqJob.Tests.ps1

```diff
@@ -81,7 +81,10 @@ Describe "BqJob-Query" {
     It "should query out of a pre-loaded table" {
```

Comment:
That's not a bad idea.  I'll use the line of code that I showed in the demo today.